### ISSUE TITLE
Marshalling big.Int

### ISSUE THREAD

**Issue by maran** [go-mgo/mgo#89]
I was trying to marshall some big.Ints into MongoDB but I guess there is no support yet since I'm just getting empty values. What would be the best way to deal with this, saving them as strings? Any advice would be appreciated.

**Comment by niemeyer** [comment]
Yeah, the struct is not special so it's being marshaled as a common document. We could probably marshal big.Int in and out as a string, though. Will consider that for the next release.
